surface_potential_analysis/state_vector/state_vector_list.py

```diff
@@ -153,8 +161,62 @@ def as_state_vector_list(
 
```

Comment:
_B2Inv is now unused - delete it (if it is still used, replace uses with _B2 as defined at top of file